### DISCUSSION TITLE
黄粱一梦之——ClassIsland 的 NTP 校时向 Windows 反向覆盖

### DISCUSSION THREAD

**Discussion by LyCecilion** [ClassIsland/ClassIsland#477] [常规]
据《枕中记》载：卢生在邯郸客店遇道士吕翁，自叹穷困。吕翁取青瓷枕让卢生睡觉，这时店主正在煮小米饭。

卢生在梦中遇到自己穿越到了 A 市 B 校 C 班。这个班的白板是可谓上世纪的 i3-3220 和 2GB RAM，神奇的校网管却想方设法都为它整上了 360 全家桶。卢生坐不住了，用 U 盘里的镜像文件为白板重装了 Windows 10，赢得了班上同学的一致好评。然而卢生结果却意外发现——每次关机之后再开机，系统的时间会保留在关机的时间上。

卢生发现 ClassIsland 的 NTP 校时比 Windows 的那个更稳定，更快速，他暂且不知道原因。如果 CI 的 NTP 校时能覆盖 Windows 的时间就好了。他一边想着，却突然惊醒——

一觉醒来，店家的小米饭还没熟。后人就根据这个故事概括出成语「黄粱一梦」，比喻虚幻的梦境和不可实现的欲望。

**Comment by HelloWRC** [comment]
这种需求建议还是用专门用来同步系统时间的工具，ClassIsland 不太适合做这种功能。

> 每次关机之后再开机，系统的时间会保留在关机的时间上。

🤔感觉这个问题我们班的电脑最近也有遇到，有时间研究下。

**Comment by jiangyin14** [comment]
你群逆天文学家

**Comment by wjj-8283** [comment]
可以选择在开机自启时使用w32tm /resync /force命令行自动校时，前提是将Windows Time服务调整为自动并启动此服务

**Comment by yixuan0826** [comment]
要不要试试改Windows的NTP？
ClassIsland用的是这个：ntp.aliyun.com

**Reply by jamyido** [reply]
感觉没有必要，阿里云的时间服务器没啥问题

**Reply by yixuan0826** [reply]
我是指改Windows的
